### ISSUE TITLE
New Windows Bing Copilot in new Windows 11 update, breaks lively wallpaper.

### ISSUE THREAD

**Issue by ianTevesAcc** [rocksdanister/lively#1911]
**Describe the bug**
New Windows Bing Copilot in new Windows 11 update, breaks lively wallpaper.

**To Reproduce**
1. Update to latest windows 11 version
2. update to latest Graphics driver for windows
3. Open windows Bing Copilot by pressing "windows + c"
4. Lively stops working and will stay that way until lively is re installed.

**Expected behavior**
Windows Bing Copilot opens with no problems.

**Screenshots/Video**
![image](https://github.com/rocksdanister/lively/assets/105651670/c394dd0c-25e3-4ff2-aa4a-5ebe98df1c1f)

**Desktop (please complete the following information):**
 - OS: Windows 11 (22621.2361)
 - Intel Graphics 31.0.101.4824
 - Mountain Rain Wallpaper

**Additional context**
Windows Bing Copilot may be the cause of this

**Log file (Important)**
[lively_log_20230928_013236.zip](https://github.com/rocksdanister/lively/files/12738673/lively_log_20230928_013236.zip)

**Comment by EyasDmour** [comment]
same on Nvidia graphics... I'm guessing this is because copilot seems to fully eat a part of the screen to show itself, instead of just showing on top of everything. which is very questionable.
I've been led to this conclusion based on desktop icons shifting for Copilot, as well as Google Chrome resizing for Copilot

Here is an exaggerated example:
![Screenshot 2023-09-27 163327](https://github.com/rocksdanister/lively/assets/72095253/43da5f8a-daee-4c20-a27f-9437b4fecb4d)
![Screenshot 2023-09-27 163336](https://github.com/rocksdanister/lively/assets/72095253/fd6fbf7d-dfed-418c-9a57-77ac7a8f5c05)

I have little actual software development experience, yet I have a feeling that this is going to be very annoying to fix...

**Comment by rocksdanister** [comment]
This is tricky for me to debug because Copilot is on rolling release and not available in India at the moment.

There is a change in insider that may break Lively, should be fine on 23H2 stable release.

Have you tried usual things like reinstalling Lively, restarting Windows after update? https://github.com/rocksdanister/lively/issues/1907

Anyhow, anyone know any way I can force enable Copilot on a test Windows setup?

**Comment by CeelB-2055440** [comment]
It might not be available in India, however, lucky there is a workaround. 
Just like in India, Windows Copilot is not available in the EU.

However, in the EU this workaround works.
You can open the runbox using WIN + R, then paste: 
microsoft-edge://?ux=copilot&tcp=1&source=taskbar
in the box and press enter. Windows Copilot will open.

You can also add a shortcut on the desktop with the same command as source.
If you assign the letter C as hotkey; you can open it using CTRL + ALT + C.

**Comment by jackkerouac** [comment]
I have the same issue in Windows 11. To temporarily fix, I just close Lively in the Taskbar and then restart. Not a great solution, but you do what you can.

**Comment by ianTevesAcc** [comment]
Yeah I don't think this issue will be solved anytime soon so I just turned off Windows Copilot through taskbar settings and restarted lively. Seems to work well for now.

**Comment by rocksdanister** [comment]
> It might not be available in India, however, lucky there is a workaround. Just like in India, Windows Copilot is not available in the EU.
> 
> However, in the EU this workaround works. You can open the runbox using WIN + R, then paste: microsoft-edge://?ux=copilot&tcp=1&source=taskbar in the box and press enter. Windows Copilot will open.
> 
> You can also add a shortcut on the desktop with the same command as source. If you assign the letter C as hotkey; you can open it using CTRL + ALT + C.

On my Windows 11 22H2 Build 22621.2361 its just opening Edge browser.

**Comment by sameert89** [comment]
I am using copilot in India. I have 2 computers and both of them got the copilot update. The same issue is happening with me as well.

**Comment by CeelB-2055440** [comment]
Here is a tutorial: https://youtu.be/iUarkBAJXmg?si=yKuecCnP5Marvfuy

**Comment by rocksdanister** [comment]
I just got the update, looks like opening the Copilot is complete resetting the desktop and rearranging the icons.

As a temporary workaround I can try resetting the wallpaper but then there will be a flicker, will think of something.

I recommend turning off copilot for the time being from the taskbar settings.

**Comment by CeelB-2055440** [comment]
I want to add that opening an application in full screen mode, like minecraft, also causes this issue.

**Comment by ianTevesAcc** [comment]
@CeelB-2055440 you can set it to open in Windowed Full Screen and it shouldn't happen anymore.

**Comment by CeelB-2055440** [comment]
Just like with windows copilot, if you start it when minecraft is open in full screen, it works. But also just like copilot, when you exit fullscreen mode, it breaks again. But not for instance mc edge. 

However, I noticed that minecraft makes the desktop fliker even without lively wallpaper. So I think that minecraft makes the screen redraw as well. Following your instructions; makes the issues resolve itself after a minute. (not for copilot) So it micht be usefull to add a button that reloads the wallpapers. But in both cases (mc started after wilvely wallpaper, and copilot ) turing the wallpaper off and on again doesn't fix it.

TL;DR
Please add a manual option as well for the copilot fix.
